addons/web/static/src/js/framework/data.js

```diff
@@ -1117,6 +1161,29 @@ function compute_domain (expr, fields) {
                 if (!_.isArray(val)) val = [val];
                 stack.push(!_(val).contains(field_value));
                 break;
+            case 'child_of':
+                if ((field_value.length == 0) && (val.length == 0)) {
+                    stack.push(true);
+                    break;
+                }
+                if (field_value.length == 0) {
+                    stack.push(false);
+                    break;
+                }
+                if (!_.isArray(val)) val = [val];
+                var itens_esquerda = field_value[0][2];
```

Comment:
same comment of odoo PR: var names in english pls :)
https://github.com/odoo/odoo/pull/15211#pullrequestreview-20060345